app.test.js

```diff
@@ -147,3 +147,40 @@ describe("/api/reviews/:review_id/comments", () => {
       });
   });
 });
+describe(" /api/reviews/:review_id/comments", () => {
+  it("POST 201: should request body accepts an object with two properties: username and body. And should respond with posted comment.", () => {
+    const newComment = { username: "philippaclaire9", body: "cool game!" };
+    return request(app)
+      .post("/api/reviews/5/comments")
+      .send(newComment)
+      .expect(201)
+      .then((res) => {
+        expect(res.body.comment).toMatchObject({
+          comment_id: expect.any(Number),
+          review_id: 5,
+          author: "philippaclaire9",
+          body: "cool game!",
+          votes: expect.any(Number),
+          created_at: expect.any(String),
+        });
+      });
+  });
+  it("GET 400: should respond with an error message indicating requested id is invalid. ", () => {
+    return request(app)
+      .post("/api/reviews/not-a-num/comments")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad Request");
+      });
+  });
+  it("GET 400: should respond with an error message if the username is not valid. ", () => {
+    const newComment = { username: "elif", body: "cool game!" };
+    return request(app)
+      .post("/api/reviews/5/comments")
+      .send(newComment)
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("The key is not present!");
+      });
+  });
```

Comment:
A couple more tests to add: 
201 or 400 (up to you): ignores unnecessary properties on the sent comment
404: non existent id
400: missing required keys